test/helper.rb

```diff
@@ -11,6 +11,8 @@
   end
 end
 
+$LOAD_PATH.unshift('test')
```

Comment:
Can we omit this? I prefer to not touch the loadpath and leave it on the user to set it up correctly.

Reply:
The load-path change was so the class would be auto-loaded by Minitest's [`load_plugin` logic](https://github.com/seattlerb/minitest/blob/6257210b7accfeb218b4388aaa36d3d45c5c41a5/lib/minitest.rb#L97-L105). Updated to manually load the plugin in e299737deac18c3d8329b227cf45a204cb8821d2.